stories/Guidelines.stories.mdx

```diff
@@ -38,3 +38,49 @@ rules. You can find all settings for Prettier in `.prettierrc`.
 
 For Installation based off your favorite code editor, please follow Prettier's
 [Editor Integration documentation](https://prettier.io/docs/en/editors.html)
+
+## Custom Assertions
+
+To help to test component, new `QUnit.assert` functions can be setup by adding the following code in `test-helper.ts`
+
+```typescript
+import * as QUnit from 'qunit';
+import registerAssertions from '@upfluence/oss-components/test-support/register-assertions';
+registerAssertions(QUnit.assert);
+```
+
+In oss-components project, all custom assertions will be store in `addon-test-support/custom-assertions`. The following code
```

Comment:
`will be stored`

but aren't they already stored ?
`all custom assertions are stored in`